uni-stark/tests/mul_air.rs

```diff
@@ -1,30 +1,47 @@
+use std::marker::PhantomData;
+
 use itertools::Itertools;
 use p3_air::{Air, AirBuilder, BaseAir};
 use p3_baby_bear::BabyBear;
 use p3_challenger::DuplexChallenger;
+use p3_commit::testing::TrivialPcs;
 use p3_commit::ExtensionMmcs;
 use p3_dft::Radix2DitParallel;
 use p3_field::extension::BinomialExtensionField;
-use p3_field::Field;
-use p3_fri::{FriConfig, TwoAdicFriPcs, TwoAdicFriPcsConfig};
+use p3_field::{AbstractField, Field};
+use p3_fri::{FriConfig, TwoAdicFriPcs};
 use p3_matrix::dense::RowMajorMatrix;
 use p3_matrix::MatrixRowSlices;
 use p3_merkle_tree::FieldMerkleTreeMmcs;
 use p3_poseidon2::{DiffusionMatrixBabybear, Poseidon2};
 use p3_symmetric::{PaddingFreeSponge, TruncatedPermutation};
-use p3_uni_stark::{prove, verify, StarkConfig, VerificationError};
+use p3_uni_stark::{prove, verify, StarkConfig, StarkGenericConfig, Val, VerificationError};
 use rand::distributions::{Distribution, Standard};
 use rand::{thread_rng, Rng};
-use tracing_forest::ForestLayer;
-use tracing_subscriber::layer::SubscriberExt;
-use tracing_subscriber::util::SubscriberInitExt;
-use tracing_subscriber::{EnvFilter, Registry};
 
 /// How many `a * b = c` operations to do per row in the AIR.
 const REPETITIONS: usize = 10;
 const TRACE_WIDTH: usize = REPETITIONS * 3;
 
-struct MulAir;
+pub struct MulAir {
+    degree: u64,
```

Comment:
Just to double check, this is just a convenience right, there shouldn't any issue with AIRs which don't have a fixed degree?

Reply:
yeah it's just to test pcs with different degrees, it's "fixed" once created